resources/sass/components/_copyright_overlay.scss

```diff
@@ -0,0 +1,11 @@
+.copyright-overlay {
+  background-color: rgba(var(--bs-white-rgb),var(--bs-bg-opacity));
+  .preview {
+    display: inline-block;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    max-width: 108px;
```

Comment:
1. in the figma design, more of the "Lorem ipsum" phrase is visible
2. also if I look at figma the spacing is bigger for the content in the modal

Reply:
the bottom spacing is not there yet